### PR TITLE
crashes: readme: add reminder to add Fixes #abcde to prs to automatically close issues.

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: tests/crashes/README.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ If you happen to fix one of the crashes, please move it to a fitting
 subdirectory in `tests/ui` and give it a meaningful name.
 Also please add a doc comment at the top of the file explaining why
 this test exists. :)
+Adding
+Fixes #12456
+Fixes #98765
+to the commit message or title of your pull request will ensure the
```
Comment (fmease) [inline tests/crashes/README.md:20]:
```suggestion
to the commit message of your pull request will ensure the
```

Putting it in the title is considered bad style cuz it's not clickable, not descriptive and may lead to ppl adding a proper title.

Furthermore, I don't think GH recognizes those in the title. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#about-linked-issues-and-pull-requests only mentions the PR description.

**Review by fmease — CHANGES_REQUESTED** [review]
one nit

File: tests/crashes/README.md (hunk 1 above)
Comment (fmease) [reply tests/crashes/README.md:20]: Ah, and putting it in the commit message is also not great because GH continuously “mentions” the issue on rebases.

**Comment by fmease** [comment]
@bors rollup
r? fmease

File: tests/crashes/README.md [hunk 2]
```diff
@@ -14,3 +14,10 @@ If you happen to fix one of the crashes, please move it to a fitting
 subdirectory in `tests/ui` and give it a meaningful name.
 Also please add a doc comment at the top of the file explaining why
 this test exists. :)
+Adding
+Fixes #12456
+Fixes #98765
```
Comment (fmease) [inline tests/crashes/README.md:19]:
```suggestion
Fixes #NNN.
Fixes #MMM.
```

Personally speaking I like placeholder like `NNN` more but whatever doesn't matter.

File: tests/crashes/README.md (hunk 1 above)
Comment (matthiaskrgr) [reply tests/crashes/README.md:20]: Ah yes sorry, I mean this  "body"/"header" or whatever this is called? (the first comment contains the commit message by default)

**Review by fmease — APPROVED** [review]

**Pull request merged**